include/onnxruntime/core/common/gpu_profiler_common.h

```diff
@@ -219,18 +219,35 @@ class GPUTracerManager {
   }
 
  protected:
-  GPUTracerManager() = default;
+  GPUTracerManager() {
+    auto this_as_derived = static_cast<TDerived*>(this);
+    uint64_t gpu_ts1, gpu_ts2, cpu_ts;
+
+    // Get the CPU and GPU timestamps to warm up
+    gpu_ts1 = this_as_derived->GetGPUTimestampInNanoseconds();
+    cpu_ts = this->GetCPUTimestampInNanoseconds();
+
+    // actual measurement of skew
```

Comment:
Do you estimate the offset between CPU and GPU timestamp?

Reply:
Yes, clarified the comment to reflect this.